README.md

```diff
@@ -0,0 +1,96 @@
+# Packetbeat
+
+Network data can tell you a detailed story of what's going on across your
+applications. Packetbeat is an open source project that is designed to provide
+real‑time analytics for web, database, and other network protocols. Whether
+you're in IT operations, security, or a business analyst, you can rapidly
+extend Packetbeat's functionality by adding new protocols, metrics, and
+analytics.
+
+## Usage
+
+Packetbeat can be added to any principal charm thanks to the wonders of being
+a subordinate charm. The following usage example will deploy the elk stack,
+so we can visualize our wire data once we've established the link between
+packetbeat and Logstash
+
+    juju deploy cs:~containers/bundle/elk-stack
+    juju deploy cs:trusty/consul
+    juju deploy cs:trusty/packetbeat
+    juju add-relation packetbeat:beats-host consul
+    juju add-relation packetbeat logstash
+
+
+### Deploying the minimal Beats formation
+
+If you do not need log buffering and alternate transforms on your data thats
+being shipped to ElasticSearch you can simply deploy the 'beats-base' bundle
+which stands up Elasticsearch, Kibana, and the three known working Beats
+subordinate services.
+
+    juju deploy cs:bundle/beats-core
+    juju deploy ubuntu
+    juju add-relation filebeat:beats-host ubuntu
+    juju add-relation topbeat:beats-host ubuntu
+    juju add-relation packetbeat:beats-host ubuntu
+
+### A note about the beats-host relationship
+
+The Beats suite of charms leverage the implicit "juju-info" relation interface
+which is special and unique in the context of subordinates. This is what allows
+us to relate the beat to any host, but may have some display oddities in the
+juju-gui. Until this is resolved, it's recommended to relate beats to their
+principal services using the CLI
+
+### Changing whats being shipped
+
+
+## Testing the deployment
```

Comment:
OO no you caught me. That needs to be filled out about configuring the charms protocols configuration bit. :scream: